src/tools/cephfs/top/cephfs-top

```diff
@@ -206,6 +229,24 @@ class FSTop(object):
             return "(%)"
         elif typ == MetricType.METRIC_TYPE_LATENCY:
             return "(s)"
+        elif typ == MetricType.METRIC_TYPE_SIZE:
+            return "(Kb)"
```

Comment:
`(kb)` is probably too small; in a large cluster it will look silly. How about MB/s?

Reply:
LGTM, will fix it.